lib/active_force/sobject.rb

```diff
@@ -27,7 +27,8 @@ class SObject
 
     class << self
       extend Forwardable
-      def_delegators :query, :not, :or, :where, :first, :last, :all, :find, :find_by, :find_by!, :sum, :count, :includes, :limit, :order, :select, :none
+      def_delegators :query, :not, :or, :where, :first, :last, :all, :find, :find!, :find_by, :find_by!, :sum, :count,
```

Comment:
Auto-formatter accidentally ran on save for this file.  Only real change is adding `:find!` to list of delegated methods.